consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,51 @@
-# Create your models here.
+import uuid
 from django.db import models
 
 
-class Question(models.Model):
+class UUIDPrimaryKeyBase(models.Model):
+    class Meta:
+        abstract = True
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+
+class SlugPrimaryKeyBase(models.Model):
```

Comment:
Do we have to use slugs as primary keys? what if two questions in different consultation have same slug?

Reply:
I think maybe UUIDs for everything?

Reply:
Now updated to all be UUIDs.